core/src/scene/styleMixer.cpp

```diff
@@ -205,9 +205,10 @@ void StyleMixer::mergeMapFieldTakingLast(const std::string& key, Node target, co
 
     for (auto it = sources.rbegin(); it != sources.rend(); ++it) {
         const auto& source = (*it)[key];
-        if (!source.IsMap()) {
+        if (!source || !source.IsMap()) {
```

Comment:
Huh. This really shouldn't change any behavior, an undefined node should return `false` for `IsMap()`. This might be a bug in yaml-cpp.